internal/cmd/gocdk/internal/static/_assets/demo/demo_blob.go

```diff
@@ -17,31 +23,32 @@ import (
 	_ "gocloud.dev/blob/s3blob"
 )
 
-// TODO(rvangent): This file is user-visible, add many comments explaining
-// how it works.
+// Package variables for the blob.Bucket URL, and the initialized blob.Bucket.
+var (
+	bucketURL string
+	bucket    *blob.Bucket
+	bucketErr error
+)
 
 func init() {
+	// Register handlers. See https://golang.org/pkg/net/http/.
 	http.HandleFunc("/demo/blob/", blobBaseHandler)
 	http.HandleFunc("/demo/blob/list", blobListHandler)
 	http.HandleFunc("/demo/blob/view", blobViewHandler)
 	http.HandleFunc("/demo/blob/write", blobWriteHandler)
-}
-
-var bucketURL string
-var bucket *blob.Bucket
-var bucketErr error
-
-func init() {
-	ctx := context.Background()
 
+	// Initialize the blob.Bucket using a URL from the environment, defaulting
+	// to an in-memory bucket provider.
```

Comment:
Can we add a _n.b_ here that if the user runs `serve`, it will empty the default mem blob? It's obvious-if-you-think-about-it but it might be confusing the first time they observe it.

Reply:
Sure, done.